salt/_states/metalk8s_drain.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
```

Comment:
I'm not sure we need this, if draining will only be done within an orchestrate we can just use a `module.run`

Reply:
Let's keep it for now, might be useful if we want pre-checks for idempotency/early failure.